jena-ontapi/src/main/java/org/apache/jena/ontapi/impl/repositories/DocumentGraphRepository.java

```diff
@@ -55,39 +54,39 @@ public DocumentGraphRepository(Supplier<Graph> factory) {
     }
 
     /**
-     * Validates the specified String is a valid URI or file path.
+     * Checks whether the specified string is a valid URI or a file path.
      *
-     * @param fileNameOrURI to validate
+     * @param uriOrFile to validate
      * @return the same string
      */
-    public static String check(String fileNameOrURI) {
-        Objects.requireNonNull(fileNameOrURI, "Null fileNameOrURI");
-        RuntimeException ex = new RuntimeException("Wrong URI: <" + fileNameOrURI + ">");
+    public static String checkLocation(String uriOrFile) {
+        Objects.requireNonNull(uriOrFile, "Null fileNameOrURI");
+        RuntimeException ex = new RuntimeException("Wrong URI: <" + uriOrFile + ">");
         String file = null;
-        if (fileNameOrURI.startsWith("file:")) {
```

Comment:
FYI: There is `IRILib` which has operations for file/URI.

These encapsulate normalizing `file:` URIs (e.g. `file:///...`).